examples/gitops-data/README.md

```diff
@@ -1,8 +1,83 @@
-## Zarf Simple gitops service Update
+# Zarf Simple GitOps Service Update
 
-This examples shows how to package images and repos to be loaded into the gitops service.  This package does not deploy anything itself, but pushes assets to the gitops service to be consumed by the gitops engine of your choice.
+This examples shows how to package images and repos to be loaded into the
+GitOps service.  This package does not deploy anything itself, but pushes
+assets to the GitOps service to be consumed by the GitOps engine of your
+choice.
 
-### Steps to use:
-1. Create a Zarf cluster as outlined in the main [README](../../README.md#2-create-the-zarf-cluster), note the git username / password output at the end
-2. Follow [step 3](../../README.md#3-add-resources-to-the-zarf-cluster) using this config in this folder
-3. Run `kubectl apply -k https://zarf-git-user:$(./zarf tools get-admin-password)@localhost/zarf-git-user/mirror__github.com__stefanprodan__podinfo//kustomize` to deploy podinfo into cluster from the gitops service
+## Demonstrated Features
+
+### Docker Image Deployment
+
+This example demonstrates using component `images` to deploy container images
+to a docker container image registry. Images provided to the `images` tag are
+uploaded to a Zarf hosted docker registry, which can be later used by
+Kubernetes manifests, or manually used as shown in this guide.
+
+### Tag-Provided Git Repository Clone
+
+Tag-provided git repository cloning is the recommended way of cloning a git
+repository for air-gapped deployment. Tag-provided clones are provided using
+the format `url.git@tag` as seen in the example with the `defenseunicorns/zarf`
+repository (`https://github.com/defenseunicorns/zarf.git@v0.12.0`).
+
+A tag-provided clone only mirrors the provided tag of the repository,
+displaying on the Gitea mirror as both the `master` branch and the tag itself.
```

Comment:
If the trunk branch of my repo is named `main` would it still show up as `master` in Gitea?

Reply:
It would have, recent changes correct this behavior.